pkgs/games/soldat-unstable/default.nix

```diff
@@ -0,0 +1,123 @@
+{ stdenv, fetchFromGitHub, fetchpatch, fpc, zip, makeWrapper
+, SDL2, freetype, physfs, openal, gamenetworkingsockets
+, xorg, autoPatchelfHook
+}:
+
+let
+  base = stdenv.mkDerivation rec {
+    pname = "soldat-base";
+    version = "unstable-2020-11-26";
+
+    src = fetchFromGitHub {
+      name = "base";
+      owner = "Soldat";
+      repo = "base";
+      rev = "e5f9c35ec12562595b248a7a921dd3458b36b605";
+      sha256 = "0qg0p2adb5v6di44iqczswldhypdqvn1nl96vxkfkxdg9i8x90w3";
+    };
+
+    nativeBuildInputs = [ zip ];
+
+    buildPhase = ''
+      sh create_smod.sh
+    '';
+
+    installPhase = ''
+      install -Dm644 soldat.smod -t $out/share/soldat
+      install -Dm644 client/play-regular.ttf -t $out/share/soldat
+    '';
+
+    meta = with stdenv.lib; {
+      description = "Soldat's base game content";
+      license = licenses.cc-by-40;
+      platforms = platforms.all;
+      inherit (src.meta) homepage;
+    };
+  };
+
+in
+
+stdenv.mkDerivation rec {
+  pname = "soldat-unstable";
+  version = "2020-11-26";
+
+  src = fetchFromGitHub {
+    name = "soldat";
+    owner = "Soldat";
+    repo = "soldat";
+    rev = "2280296ac56883f6a9cad4da48025af8ae7782e7";
+    sha256 = "17i3nlhxm4x4zx00i00aivhxmagbnyizxnpwiqzg57bf23hrvdj3";
+  };
+
+  nativeBuildInputs = [ fpc makeWrapper autoPatchelfHook ];
+
+  buildInputs = [ SDL2 freetype physfs openal gamenetworkingsockets ];
+  runtimeDependencies = [ xorg.libX11 ];
+
+  patches = [
+    # fix an argument parsing issue which prevents
+    # us from passing nix store paths to soldat
+    (fetchpatch {
+      url = "https://github.com/sternenseemann/soldat/commit/9f7687430f5fe142c563b877d2206f5c9bbd5ca0.patch";
+      sha256 = "0wsrazb36i7v4idg06jlzfhqwf56q9szzz7jp5cg4wsvcky3wajf";
+    })
+  ];
+
+  postPatch = ''
+    for f in client/Makefile server/Makefile; do
+      # unportable uname invocation
+      substituteInPlace "$f" --replace "uname -p" "uname -m"
+      # broken rpath settings
+      substituteInPlace "$f" --replace "-k-rpath -k'\\\$\$ORIGIN/'" ""
```

Comment:
```suggestion
      # fix unportable uname invocation and broken rpath settings
      substituteInPlace "$f" \
        --replace "uname -p" "uname -m" \
        --replace "-k-rpath -k'\\\$\$ORIGIN/'" ""
```

Reply:
The second invocation is actually not necessary due to `autoPatchelfHook`, removing.